doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+          `Docker Desktop for MacOS <https://docs.docker.com/desktop/install/mac-install/>`_.
+
+       #. **(On Windows)** When prompted for **Use WSL2 instead of Hyper-V (recommended)**, **UNTICK** this checkbox.
+          Hyper-V must be enabled to run Windows Docker containers - which you might be interested in doing in the future.
+
+       #. Once the installation finishes, restart your machine, and start Docker Desktop.
+
+Now that your Docker engine supports running Linux Docker containers, you can build or install
+the PyAnsys Geometry image.
+
+Build or install the Geometry Service image
+-------------------------------------------
+
+There are two options for users to install the PyAnsys Geometry image:
+
+* Downloading it from the :ref:`GitHub Container Registry <ref_linux_docker_ghcr>`.
+* :ref:`Building the Geometry Service Linux container <ref_linux_docker_fromscratch>`.
+
+.. _ref_linux_docker_ghcr:
+
+GitHub Container Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    This option is only available for users with write access to the repository or
+    part of the Ansys organization.
+
+Once Docker is installed on your machine, follow these steps to download the Linux Docker
+container for the Geometry service and install this image.
+
+#. Using your GitHub credentials, download the Docker image from the `PyAnsys Geometry repository <https://github.com/ansys/pyansys-geometry>`_
+   on GitHub.
+
+#. Use a GitHub personal access token with permission for reading packages to authorize Docker
+   to access this repository. For more information, see `Managing your personal access tokens
+   <https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens>`_
+   in the GitHub documentation.
+
+#. Save the token to a file with this command:
+
+   .. code-block:: bash
+
+       echo XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX > GH_TOKEN.txt
+
+#. Authorize Docker to access the repository. To see the commands to run, click the tab for your OS.
+
+   .. tab-set::
+
+       .. tab-item:: Linux/Mac
+
+           .. code-block:: bash
+
+               GH_USERNAME=<my-github-username>
+               cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
+
+       .. tab-item:: Powershell
+
+           .. code-block:: pwsh
+
+               $env:GH_USERNAME=<my-github-username>
+               cat GH_TOKEN.txt | docker login ghcr.io -u $env:GH_USERNAME --password-stdin
+
+       .. tab-item:: Windows CMD
+
+           .. code-block:: bash
+
+               SET GH_USERNAME=<my-github-username>
+               type GH_TOKEN.txt | docker login ghcr.io -u %GH_USERNAME% --password-stdin
+
+
+#. Pull the Geometry service locally using Docker with a command like this:
+
+   .. code:: bash
+
+      docker pull ghcr.io/ansys/geometry:linux-latest
+
+.. _ref_linux_docker_fromscratch:
+
+Building the Geometry Service Linux container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
Build the Geometry service Linux container
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Again, using the simple verb form in headings is preferred. Also, the recommendation is not to have more than 3 adjectives before a noun--and here we have four, which is hard to read. Can we rewrite the title as "Build the Linux container for the Geometry service"?

Reply:
Yes, I will make all the changes necessary to avoid the -ing form =)